packages/contracts/contracts/voting/majority/MajorityVotingBase.sol

```diff
@@ -426,13 +451,11 @@ abstract contract MajorityVotingBase is
             // Early execution
             return
                 proposal_.parameters.votingMode == VotingMode.EarlyExecution &&
-                worstCaseSupport(_proposalId) > proposal_.parameters.supportThreshold &&
-                participation(_proposalId) >= proposal_.parameters.minParticipation;
+                isSupportThresholdReachedEarly(_proposalId) &&
+                isMinParticipationReached(_proposalId);
```

Comment:
I would rewrite in simpler/clearer steps like:

```
if(bad) return false;
else if(bad2) return false;
else if(bad3 return false;

return true;
```

Reply:
Done in db19f844d09825c4bdd60678679bf19d0c7be262